bin/webpack-dev-server.js

```diff
@@ -331,9 +331,12 @@ function processOptions(wpOpt) {
 
 	if(argv["open"] || argv["open-page"]) {
 		options.open = true;
-		options.openPage = argv["open-page"] || "";
```

Comment:
No, what is happening is that this statement isn't hit at all, because argv is only for command line arguments.

In the case of a config file, argv will be mostly empty but options.open and options.openPage may already have been populated by the input config file, so we STILL need to set openPage to empty string regardless of how options.open got set to true.